src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,178 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+
+    private Node<T> head;
+    private Node<T> tail;
+    private int size = 0;
+
+    public MyLinkedList() {
+        head = null;
+        tail = null;
+    }
+
+    public static class Node<T> {
```

Comment:
This inner class is only for your MyLinkedList and should not be accessible from outside. Make it private.
Move this inner private class to the bottom of your MyLinkedList class
```suggestion
    private static class Node<T> {
```